main.go

```diff
@@ -605,7 +614,12 @@ func makeIngressChanges(cf *Cloudflare, kubeClientset *kubernetes.Clientset, ing
 			ingress.Annotations[annotationCloudflareState] = string(cloudflareStateByteArray)
 
 			// update ingress, because the state annotations have changed
-			_, err = kubeClientset.NetworkingV1beta1().Ingresses(ingress.Namespace).Update(ingress)
+			_, err = kubeClientset.NetworkingV1().Ingresses(ingress.Namespace).Update(ctx, ingress, metav1.UpdateOptions{
+				TypeMeta:        metav1.TypeMeta{},
+				DryRun:          []string{},
```

Comment:
Do we need to explicitly define DryRun? and do we want modifications should not be persisted?

Reply:
Removed.